Description: When building from a source tarball (as opposed to a git checkout) there is no last commit position
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Forwarded: not-needed

--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -121,8 +121,6 @@ def main(argv):
     cmd.append('--debug')
   subprocess.check_call(cmd)
 
-  shutil.copy2(
-      os.path.join(BOOTSTRAP_DIR, 'last_commit_position.h'), gn_build_dir)
   cmd = [ninja_binary, '-C', gn_build_dir, '-w', 'dupbuild=err', 'gn']
   if options.jobs:
     cmd += ['-j', str(options.jobs)]
